ui/src/app/edge/live/chpsoc/chpsoc.component.html

```diff
@@ -0,0 +1,46 @@
+<ng-container *ngIf="service.currentEdge | async as edge">
+    <ng-container *ngIf="edge.currentData | async as currentData">
+        <ion-card>
+            <ion-item lines="full" color="light">
+                <ion-avatar slot="start">
+                    <ion-icon name="flame" style="font-size: 40px" font></ion-icon>
```

Comment:
`style` should be generally avoided; is it really required here? For ion-card headers I usually take the example by @fabianfnc - something like this:

```
<ion-card>
  <ngx-loading [show]="!(edge.currentData | async)"></ngx-loading>

  <ion-item lines="full" color="light">
    <ion-icon slot="start" size="large" color="primary" name="power"></ion-icon>
    <ion-label>XXX</ion-label>
  </ion-item>

  <ion-card-content *ngIf="(edge.currentData | async).channel as channel">
    ...
  </ion-card-content>
</ion-card>
```

https://github.com/OpenEMS/openems/blob/develop/ui/src/app/edge/live/fixdigitaloutput/fixdigitaloutput.component.html#L5